homeassistant/components/spotify/media_player.py

```diff
@@ -90,6 +100,16 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         oauth, config.get(CONF_NAME, DEFAULT_NAME), config[CONF_ALIASES])
     add_entities([player], True)
 
+    def play_playlist_service(service):
+        media_content_id = service.data.get(ATTR_MEDIA_CONTENT_ID)
```

Comment:
Please use `dict[key]` for required schema keys and keys with default schema values.

Reply:
I'm in holidays, i will try sort it Sunday night when I'm back. Thanks

Reply:
improved!